docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing a node
+
+Right now 4 cores and 8 GB of RAM should be enough to run a node, but it might increase in the future. More info in the [FAQ](docs/faq.md).
+
+## From binaries
+If you just wish to run a Massa node without compiling it yourself, you
+can simply run the latest binary (download and execute):
+
+-   [Windows
+    executable](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-windows)
+-   [Linux
+    binary](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-linux)
+-   [MacOS
+    binary](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-darwin)
+
+## From source code
+Otherwise, if you wish to run a Massa node from source code, here are the steps to follow:
+
+### On Ubuntu / MacOS
+
+-   on Ubuntu, these libs must be installed:
+    `sudo apt install pkg-config curl git build-essential libssl-dev`
```

Comment:
Why do you install these libs through curl ?
These two should be enough
`sudo apt install build-essential`
`sudo apt install libssl-dev`

Reply:
* git is necessary to install from source.
* pkg-config is necessary for builds to succeed on some systems (this was found by trial + error)
* curl is useful for debugging, when we ask people to try some commands